### PR TITLE
[UPDATE] Check ProccessActive

### PR DESCRIPTION
**Pull request by ramonornela**
- Support array chek proccess.

### CODE REVIEW

**Comment by ramonornela** [comment]
Improvement verification of multiple proccess, ticket similar https://github.com/liip/LiipMonitorBundle/issues/32 created by @lsmith77, request this to more checks.

**Comment by ramonornela** [comment]
ping @lsmith77

**Comment by lsmith77** [comment]
here again .. it would be great if you could add a test.

File: Check/ProcessActiveCheck.php [hunk 1]
```diff
@@ -41,6 +46,6 @@ public function check()
      */
     public function getName()
     {
-        return 'Process Active: ' . $this->command;
+        return 'Process Active';
```
Comment (lsmith77) [inline Check/ProcessActiveCheck.php:49]: its a but unfortunately to loose any semantic information in the name. maybe we need a "name" setting to still be able to identify the command "group". at any rate it should be `Processes` now .. also at the very least maybe we should then add a`count()`` of the commands to the string.

**Comment by kbond** [comment]
fixed in liip/LiipMonitorBundle#48

**Pull request closed without merging**